Library/Formula/lsyncd.rb

```diff
@@ -56,28 +54,23 @@ class Lsyncd < Formula
     end
   end
 
-  # patch for CVE-2014-8990
-  # https://github.com/axkibe/lsyncd/commit/e9ffda07f0145f50f2756f8ee3fb0775b455122b
-  # https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2014-8990
-  patch do
-    url "https://gist.githubusercontent.com/tdsmith/d807811d3c6965b0221e/raw/965545662eec39b60d50645487e6ade9d7d43834/cve-2014-8990.diff"
-    sha256 "c6476855acaefd4619bd6900751247c2af12983ed2aff9bdfbf971ffcb662fc2"
-  end
-
   def install
     # XNU Headers
     resource("xnu").stage buildpath/"xnu"
-    ENV.append "CPPFLAGS", "-Ixnu"
 
-    ENV.append "XML_CATALOG_FILES", "#{Formula["docbook"].opt_prefix}/docbook/xml/4.5/catalog.xml"
-    ENV["A2X"] = "#{Formula["asciidoc"].opt_bin}/a2x"
+    args = std_cmake_args
+    args << "-DWITH_INOTIFY=OFF"
+    args << "-DWITH_FSEVENTS=ON"
+    args << "-DXNU_DIR=#{buildpath/"xnu"}"
+
+    # DESTINATION bin
+    inreplace "CMakeLists.txt", "DESTINATION bin",
```

Comment:
I was not able to find such a flag in this build. Maybe someone more experienced in cmake can take a look. If not then we can pull this.